parts/k8s/kubernetesinstalls.sh

```diff
@@ -187,6 +186,7 @@ downloadAzureCNI() {
 }
 
 downloadContainerd() {
+    CONTAINERD_DOWNLOAD_URL="${CONTAINERD_DOWNLOAD_URL_BASE}cri-containerd-${CONTAINERD_VERSION}.linux-amd64.tar.gz"
```

Comment:
we're already setting `CONTAINERD_DOWNLOAD_URL` in the VHD script itself so this will override it

Reply:
no longer setting `CONTAINERD_DOWNLOAD_URL` in the VHD script